dts/bindings/usb/usb.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2018, I-SENSE group of ICCS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: USB Base Structure
+id: USB
+version: 0.1
+
+description: >
+    This binding gives the base structures for all USB devices
+
+inherits:
+    !include usb-ep.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+
+    clock-frequency :
+      type: int
+      category: optional
+      description: Initial clock frequency in Hz
+      generation: define
+
+    clocks:
```

Comment:
Should this be in the base usb.yaml definition or in the SoC specific files?

Reply:
I thought that I have removed clocks property. I'll remove it.